docs/src/index.rst

```diff
@@ -2,7 +2,7 @@
 Source-Code
 ===========
 
-This is a partial documentation of our source code.  We are not aim to document
+This is a partial documentation of our source code. We are not aiming to document
```

Comment:
The double space after the dot of sentence end was intended. Typographic tools use the two spaces after the end of a sentence to distinguish from the period of an abbreviation.